python/py-kaptan/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-kaptan
+version             0.5.10
+maintainers         {egorenar @egorenar} openmaintainer
```

Comment:
FYI: the first field `egorenar` indicates the email address egorenar@macports.org. Looks like you're not a MacPorts member yet? If so you should use your full email address.

See the "maintainers" section in https://guide.macports.org/#reference.keywords for details.

Reply:
thanks, fixed.